components/vf-tabs/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 2.0.0-rc.1
+
+* updates the styling to match the design direction.
```

Comment:
If I follow, this is a no-code-change needed migration that is mainly "visually breaking"?

Reply:
Yeah - potentially we could do a minor release -- but as it was such a big stylistic change I thought going with a major release would stop any products using it have a sudden 'wtf?' if they use dependabot etc etc

Reply:
Cool. figured. I'll dive in and a note to the changelog about this — as I've already pinged you with enough things.